tests/integration/devfile/cmd_devfile_push_test.go

```diff
@@ -552,3 +574,13 @@ var _ = Describe("odo devfile push command tests", func() {
 	})
 
 })
+
+// Wait for the session stdout output to contain a particular string
+func waitForOutputToContain(substring string, session *gexec.Session) {
+
+	Eventually(func() string {
+		contents := string(session.Out.Contents())
+		return contents
+	}, 180, 10).Should(ContainSubstring(substring))
+
+}
```

Comment:
this should be in the /tests/helper package

Reply:
Done.